docs/get-involved/join-the-drips-community.md

```diff
@@ -3,9 +3,9 @@ id: join-the-drips-community
 title: Join the Drips Community
 ---
 
-The Drips community is a part of the greater [Radicle community][rc].
+The Drips community is a part of the [Radicle community][rc].
```

Comment:
Link seems broken.

Reply:
This page doesn't exist on the Radicle Docs site anymore, so removing the link entirely for now.